samples/amcache.py

```diff
@@ -213,6 +218,19 @@ def main(argv=None):
         w.writerow(["timestamp", "timestamp_type", "path", "sha1"])
         for e in sorted(entries, key=lambda e: e.timestamp):
             w.writerow([e.timestamp, e.type, e.entry.path, e.entry.sha1])
+
```

Comment:
probably should add check to see that both `-j` and `-t` are not both provided. i assume the user would figure out pretty quickly on their own, but its best to be explicit.